Assets/Scripts/Actors/ActorComponents/PlayerActorResources.cs

```diff
@@ -174,7 +174,7 @@ void OnTriggerEnter( Collider other )
 			PickupResource( resourceComponent.resourceData );
 
 			Destroy( other.gameObject );
-			WadeUtils.TempInstantiate( resourcePopPrefab, other.transform.position, Quaternion.identity, 1f );
+			SoundManager.instance.Play3DSoundAtPosition( resourcePopAudio, other.transform.position );
```

Comment:
This is my preferred way of doing it and I'd argue it makes the singleton code way more readable. Since that's been the driving factor in defining our code standard I think we should keep it this way.